### PR TITLE
chore: improve "please reach out" section of Black Lives Matter page

### PR DESCRIPTION
**Pull request by Trott**
Edit the sentences for bervity and clarity. Also, added myself to the
list of Twitter handles. Expanded "TSC" to "Technical Steering Committee" because I imagine a lot of people will have no idea what "TSC" stands for.

### CODE REVIEW

**Comment by Trott** [comment]
By the way, any reason the display for `li` in this section is `inline` rather than `list-item`? I don't find the compressed display a practical or aesthetic improvement over the default but I obviously missed the memo on why this was done so maybe there are considerations I'm ignorant of at the moment.

**Comment by MylesBorins** [comment]
Didn't want it too take away from the impact of the list of names at the top of the page

**Comment by Trott** [comment]
> Didn't want it too take away from the impact of the list of names at the top of the page

Got it! Just to be clear: This doesn't change that styling. I was just asking about it. (Implied question: Can I get an approval on this?)

File: layouts/black-lives-matter.hbs [hunk 1]
```diff
@@ -289,7 +289,13 @@
   </section>
 
   <section id="we-are-here-for-you">
-      <p>Please feel welcome to reach out to the <a href="https://github.com/nodejs/node#tsc-technical-steering-committee">Node.js TSC members</a> and <a href="https://github.com/nodejs/community-committee#community-committee-members">Node.js Community Committee members</a> via our email addresses listed in the links, respectively, or via Twitter DMs to the following folks in the project:</p>
+      <p>
+        Please feel welcome to contact
+        <a href="https://github.com/nodejs/node#tsc-technical-steering-committee">Technical Steering Committee</a>
+        and <a href="https://github.com/nodejs/community-committee#community-committee-members">Community Committee</a>
+        members. The preceding links contain email addresses for all members. If
+        you prefer, contact any of the following members on Twitter:
```
Comment (MylesBorins) [inline layouts/black-lives-matter.hbs]:
```suggestion
        you prefer, you can contact any of the following members on Twitter:
```

Read a bit strange but maybe I'm over thinking it

**Review by MylesBorins — APPROVED** [review]
LGMT with a tiny nit

**Pull request merged**